blurring_as_a_service/performace_evaluation_pipeline/submit_performance_evaluation_pipeline.py

```diff
@@ -0,0 +1,96 @@
+from typing import Dict
+
+from azure.ai.ml import Input, Output
+from azure.ai.ml.constants import AssetTypes
+from azure.ai.ml.dsl import pipeline
+
+from blurring_as_a_service.performace_evaluation_pipeline.components.evaluate_with_coco import (
+    evaluate_with_coco,
+)
+from blurring_as_a_service.performace_evaluation_pipeline.components.evaluate_with_cvt_metrics import (
+    evaluate_with_cvt_metrics,
+)
+from blurring_as_a_service.performace_evaluation_pipeline.components.validate_model import (
+    validate_model,
+)
+from blurring_as_a_service.settings.flags import PipelineFlag
+from blurring_as_a_service.settings.settings import BlurringAsAServiceSettings
+from blurring_as_a_service.utils.aml_interface import AMLInterface
+
+
+@pipeline()
+def performance_evaluation_pipeline(
+    validation_data, annotations_json, yolo_yaml_path, yolo_validation_output
+):
+    performance_evaluation_flags = BlurringAsAServiceSettings.get_settings()[
+        "performance_evaluation_pipeline"
+    ]["flags"]
+
+    if performance_evaluation_flags & PipelineFlag.VALIDATE_MODEL:
```

Comment:
Maybe the flag for validate the model we can skip it since it's the main purpose of the pipeline?
Otherwise it can also not be run. What was the idea behind it?